tests/python_tests/test_vlm_pipeline.py

```diff
@@ -480,7 +480,7 @@ def streamer(subword):
 
 
 def retry(func):
```

Comment:
Maybe it is better to move `max_retries` to kwargs?
```python
def retry(func, max_retries=20):
    ...
```

Reply:
No one uses other value - YAGNI